data/tlg4170/tlg001c/__cts__.xml

```diff
@@ -0,0 +1,7 @@
+<ti:work xmlns:ti="http://chs.harvard.edu/xmlns/cts" groupUrn="urn:cts:greekLit:tlg4170" xml:lang="grc" urn="urn:cts:greekLit:tlg4170.tlg001c.1st1K-grc1">
+  <ti:title xml:lang="lat">Vita metrica</ti:title>
+  <ti:edition urn="urn:cts:greekLit:tlg4170.tlg001c.1st1K-grc1" workUrn="urn:cts:greekLit:tlg4170.tlg001c.1st1K-grc1">
```

Comment:
`workUrn` attribute here should be the work-level URN and not the edition-level URN. So it should be `urn:cts:greekLit:tlg4170.tlg001c`